src/utils/GetStyledCalendarElement/GetStyledCalendarElement.js

```diff
@@ -76,3 +76,51 @@ export const header = (totalContributions, isLoading) => {
 
   return calendarHeader;
 };
+
+export const tooltip = () => {
+  const calendarTooltip = document.createElement('DIV');
+
+  calendarTooltip.id = 'tooltip';
+
+  calendarTooltip.style.background = 'rgba(0, 0, 0, 0.8)';
+  calendarTooltip.style.borderRadius = '3px';
+  calendarTooltip.style.fontSize = '12px';
+  calendarTooltip.style.padding = '10px';
+  calendarTooltip.style.textAlign = 'center';
+  calendarTooltip.style.position = 'absolute';
+  calendarTooltip.style.zIndex = '99999';
```

Comment:
How about a just high enough number? :)

Reply:
Actually, I copied the same styles that GH uses. :)

But that's right.

Reply:
It seems that we don't even need the `z-index` property.